### DISCUSSION TITLE
disable Versionnumber on Loginpage

### DISCUSSION THREAD

**Discussion by Rimos911** [dani-garcia/vaultwarden#5218] [Ideas]
Hello,

could you please add an option to disable the Versionnumber of the Vaultwarden Server on the Main-Loginscreen.

Yes, the Admin has to keep the Server up-to-date, but if an Attacker see that the Server Version is old, he can investigate potential vulnerabilities of the Server an can start his attack.

As I said, only as an option for the admin (enable/disable: Version)

Thank you!

example:

![eabled](https://github.com/user-attachments/assets/c2ea045c-a102-42d4-ac3f-85c2c5242dc2)

![disabled](https://github.com/user-attachments/assets/f1014ffc-5343-436b-9363-87a8b95a6f55)

**Comment by BlackDex** [comment]
No. This is not something we will implement as there are several other ways to check and verify which version there is running.
It has no real benefit of leaving those details out.

If you really want this you need to build the web-vault your self and remove this.

**Reply by mikailyoelek** [reply]
@BlackDex this has nothing to do with real benefit. It shouldn't be possible for an attacker, to discover the running version of the application so easily. In my point of view this should be only visible to logged in users and not to everyone!!!

**Reply by BlackDex** [reply]
Not possible, and this is just own Bitwarden works. It also needs `/api/config` where the version is mentioned which is accessible by everyone since it needs that before any auth is done at all.

**Reply by mikailyoelek** [reply]
@BlackDex wow, what a quick response, now it makes sense ... but still, this seems an awkward design decision for discovery. Still thank you for clarification 👍
